frontend/catalyst/passes/builtin_passes.py

```diff
@@ -950,53 +950,60 @@ def ppm_compilation(
 
     **Example**
 
-    The ``commute_ppr`` compilation pass can be applied as a dectorator on a QNode:
+    If a merging resulted in a PPM acting on more than
+    ``max_pauli_size`` qubits (here, ``max_pauli_size = 2``), that merging would be skipped.
+    However, when decomposed into PPMs, at least one qubit will be applied, so the final
+    PPMs will act on at least one additional qubit.
 
     .. code-block:: python
 
         import pennylane as qml
-        from functools import partial
+        from catalyst import qjit, measure
+        from catalyst.passes import ppm_compilation
 
-        qml.capture.enable()
+        pipeline = [("pipe", ["enforce-runtime-invariants-pipeline"])]
```

Comment:
pipeline = [("pipe", ["quantum-compilation-stage"])]

change this and other occurrences.